kernel/standalone/src/kernel.rs

```diff
@@ -63,7 +63,7 @@ impl Kernel {
         .unwrap();
 
         let stdout_module = redshirt_core::module::Module::from_bytes(
-            &include_bytes!("../../../modules/target/wasm32-wasi/release/x86-stdout.wasm")[..],
+            &include_bytes!("../../../modules/target/wasm32-wasi/release/arm-stdout.wasm")[..],
```

Comment:
This needs to be figured out before merging. I guess I could add some `#[cfg]`s, but it's really not great.
The proper way to do that is related to how one would define a target.